src/components/resumes/Details.jsx

```diff
@@ -56,6 +77,15 @@ function Details() {
     }
   };
 
+  const getSkills = async (event) => {
+    console.log("Skill input: ", skillInput);
+    event.preventDefault();
+    const response = await getSkillsByName(skillInput);
+    console.log(response);
+  };
+
+  const [skillInput, setSkillInput] = useState("");
+
```

Comment:
You con move this useState to group useState just in a place.

Reply:
Done